app/notifications/process_notifications.py

```diff
@@ -148,11 +148,8 @@ def persist_notification(
 
 
 def send_notification_to_queue_detached(
-    key_type, notification_type, notification_id, research_mode, queue=None
+    notification_type, notification_id, queue=None
 ):
-    if research_mode or key_type == KEY_TYPE_TEST:
```

Comment:
We want to keep the test API key here, though I'm not certain how this should be revised. Happy to discuss more if it's not clear to you, either.